app/module/test-behaviors/Kconfig

```diff
@@ -0,0 +1,3 @@
+config ZMK_TEST_BEHAVIORS
+    bool "Include behaviors used for testing purposes"
+    default n
```

Comment:
Superfluous, this is the default, and adding it can actually complicate some scenarios. See https://docs.zephyrproject.org/latest/build/kconfig/tips.html#redundant-defaults